app/helpers/tag_lists_helper.rb

```diff
@@ -15,7 +10,20 @@ def available_tags_checkboxes(tags)
     end, '')
   end
 
-  private
+  def format_tag_category(category)
+    case category
+    when :other
+      t('.category_other')
+    when :category_validation
+      t('.category_validation')
+    else
+      category
+    end
```

Comment:
Consider using a hash for this mapping, like you did in PersonTags::Translator

Reply:
I don't like having to much methods in helper modules, since all methods are exposed anyway.